opensearch-dashboards-2.yaml

```diff
@@ -0,0 +1,83 @@
+package:
+  name: opensearch-dashboards-2
+  version: 2.11.1
+  epoch: 0
+  description: Open source visualization dashboards for OpenSearch
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - nodejs-18
+
+environment:
+  contents:
+    packages:
+      - apk-tools
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gcc-12
+      - gcc-12-default
+      - git
+      - node-gyp
+      - nodejs-18
+      - py3-setuptools
+      - python3
+      - xorg-server
+      - yarn
+  environment:
+    CC: gcc
+    CXX: g++
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
+      tag: ${{package.version}}
+      expected-commit: 989d8f41f37cca3275bf3fedc5c2057a717d1d64
+
+  - runs: |
+      # Workaround for "OpenSearch Dashboards should not be run as root.  Use --allow-root to continue."
+      # This change will add the --allow-root when running the build_ts_refs and register_git_hook scripts
+      sed -i 's/\("osd:bootstrap": "scripts\/use_node scripts\/build_ts_refs\)\( && scripts\/use_node scripts\/register_git_hook\)/\1 --allow-root\2 --allow-root/' package.json
+
+  - runs: |
+      set -x
+
+      # Our commond LDFLAGS cause some issues when building for aarch64. We
+      # unset our global flags to allow the build to succeed.
+      unset LDFLAGS
+      yarn osd bootstrap --allow-root
+      yarn build-platform --linux-arm --skip-os-packages --release --allow-root
```

Comment:
I'm wondering if these flags need to change based on which arch is building. I.e x86 or arm. 

I can take a look shortly